code/src/components/MoviePage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState} from 'react';
+import {  useParams } from 'react-router-dom';
+
+import './moviePage.css'
+
+export const MoviePage = () => {
+    const { id } = useParams();
+    const [movie, setMovie] = useState({});
+
+    useEffect(() => {
```

Comment:
When doing the useEffect you can pass in the id and then write in the [ ] on line 16 id. As far as I understand it it'll help the useEffect to be performed to 100% when the component is mounted. You can write it like this:
` useEffect((id) => { 
        fetch(`https://api.themoviedb.org/3/movie/${id}? 
   api_key=0cb608022c0a0fbef75881f68133d694&language=en-US`)
        .then((res) => res.json())
        .then((json) => {
            setMovie(json)
        });
    }, [id]);`